libcontainer/selinux/selinux.go

```diff
@@ -275,7 +278,9 @@ func mcsAdd(mcs string) error {
 }
 
 func mcsDelete(mcs string) {
+	mcsLock.Lock()
 	mcsList[mcs] = false
+	mcsLock.Unlock()
```

Comment:
In line https://github.com/opencontainers/runc/pull/735/files#diff-0ff20b2f0ffb08e6b40c79c94e4c8e65R283 `defer` is being used to unlick. Maybe it'd be nice to have a consistent way of unlocking?